packages/forms-web-app/src/validators/custom/clamav.js

```diff
@@ -0,0 +1,26 @@
+const NodeClam = require('clamscan');
+const fs = require('fs');
+
+module.exports = async (fileInformation, errorMessage) => {
+  if (typeof fileInformation !== 'undefined') {
+    if (typeof fileInformation.tempFilePath !== 'undefined') {
```

Comment:
you could reduce the nesting here by using optional chaining:

`if (fileInformation?.tempFilePath) {`

same meaning as lines 5 & 6

![Screenshot from 2021-05-25 09-46-19](https://user-images.githubusercontent.com/6329541/119468122-130a5b80-bd3e-11eb-8761-d6cc8aafff11.png)